src/reference/modules/usage/pages/configuration_management.adoc

```diff
@@ -305,19 +306,113 @@ The main goal of Rudder is to compute compliance of configured rules on nodes by
 
-On extreme cases, for example for embedded hardware with costly network data plan, you may want to totally disable reporting. In that case, 
+On extreme cases, for example for embedded hardware with costly network data plan, you may want to totally disable reporting. In that case,
 Rudder will assume that the node is doing what it needs and Rudder will not put a "no answer" status for it, even if it does not get reports back.
 
-The setting is available either globally in the `Settings` menu (see image below), or on a node-by-node basis in its setting tab. 
+The setting is available either globally in the `Settings` menu (see image below), or on a node-by-node basis in its setting tab.
 
 We strongly advise to keep the default configuration and use the `Full compliance` mode.
 
 image:assets/images/reporting-mode-setting.png[Global configuration of reporting mode]
+
+[[_keep_last_compliance_duration]]
+== Non-answering nodes and configuration of last compliance validity
+
+When a node is not sending compliance reports to Rudder, it can be a big problem, like when a
+server goes outline. And even if Rudder is not a monitoring tool, in that case, you want to make very
+clear the fact that server didn't call home recently.
+To address that topic, by default, Rudder considers that after 2 times the duration of the agent run period,
+the node is not answering and all the compliance is put in "no answer" which is considered to be an error with
+regard to compliance.
+
+Sometime, it's totally expected that nodes often go outline, like in the case of managing user laptops.
+In that case, you may want to change Rudder behavior to keep last known compliance for much longer
+than 2 agent run periods, perhaps even as long as several days.
+
+This configuration is done through a sub-property of the system node property `rudder`.
+For example, to keep the last compliance for one hour, go to the node's properties tab, find `rudder`
+property (likely marked `inherited` if you didn't change it yet), click "edit" action:
+
+image:assets/images/edit_rudder_system_node_property.png[Edit rudder node property]
+
+Then, add the following new sub-properties in the JSON value:
+
+[source, json]
+----
+Property name: rudder
+Type: JSON
+Value:
+{
+  {
+... // don't change other existing sub-properties
+  },
+  "compliance_expiration_policy": {
+     "mode":"keep_last",
+     "duration":"1 hour"
+  }
+}
+----
+
+image:assets/images/set_compliance_expiration_policy.png[Change node compliance expiration policy]
+
+Save, and now the node can be offline for one hour and you still have access to its last compliance with an adapted explanation:
+
+image:assets/node_keep_last_compliance_view.png[Compliance kept for a non-answering node]
+
+=== Configure expiration policy
+
+The `rudder -> compliance_expiration_policy` is almost a standard node property: it can be configured at group levels
```

Comment:
The arrow size is really small in the doc, maybe JSON path is clearer ?
```suggestion
The `rudder.compliance_expiration_policy` is almost a standard node property: it can be configured at group levels
```